samples/cpp/ngraph_function_creation_sample/main.cpp

```diff
@@ -252,147 +224,103 @@ int main(int argc, char* argv[]) {
+        const ov::Shape output_shape = model->output().get_shape();
+        OPENVINO_ASSERT(output_shape.size() == 2, "Incorrect output dimensions for LeNet");
+
+        const auto classCount = output_shape[1];
+        OPENVINO_ASSERT(classCount <= LENET_NUM_CLASSES, "Incorrect number of output classes for LeNet model");
 
         // -------- Step 3. Apply preprocessing --------
         const Layout tensor_layout{"NHWC"};
 
         // apply preprocessing
-        auto proc = ov::preprocess::PrePostProcessor(model);
+        ov::preprocess::PrePostProcessor ppp = ov::preprocess::PrePostProcessor(model);
+
         // 1) InputInfo() with no args assumes a model has a single input
-        auto& input_info = proc.input();
+        ov::preprocess::InputInfo& input_info = ppp.input();
         // 2) Set input tensor information:
         // - layout of data is 'NHWC'
         // - precision of tensor is supposed to be 'u8'
         input_info.tensor().set_layout(tensor_layout).set_element_type(element::u8);
         // 3) Here we suppose model has 'NCHW' layout for input
         input_info.model().set_layout("NCHW");
+
         // 4) Once the build() method is called, the preprocessing steps
         // for layout and precision conversions are inserted automatically
-        model = proc.build();
-
-        // -------- Step 4. Read input images --------
-
-        const auto input = model->input();
-
-        auto input_shape = input.get_shape();
-        const size_t width = input_shape[layout::width_idx(tensor_layout)];
-        const size_t height = input_shape[layout::height_idx(tensor_layout)];
-
-        std::vector<std::shared_ptr<unsigned char>> imagesData;
-        for (auto& i : images) {
-            FormatReader::ReaderPtr reader(i.c_str());
-            if (reader.get() == nullptr) {
-                slog::warn << "Image " + i + " cannot be read!" << slog::endl;
-                continue;
-            }
+        model = ppp.build();
 
-            if (reader->size() != width * height) {
-                throw std::logic_error("Not supported format. Only MNist ubyte images supported.");
-            }
+        // Set batch size using images count
+        const size_t batch_size = digits.size();
 
-            // Store image data
-            std::shared_ptr<unsigned char> data(reader->getData(width, height));
-            if (data.get() != nullptr) {
-                imagesData.push_back(data);
-            }
-        }
-
-        OPENVINO_ASSERT(!imagesData.empty(), "Valid input images were not found");
-
-        // -------- Step 4. Reshape a model --------
+        // -------- Step 4. Reshape a model to new batch size --------
         // Setting batch size using image count
-        const auto batch_size = static_cast<int64_t>(imagesData.size());
         ov::set_batch(model, batch_size);
         slog::info << "Batch size is " << std::to_string(batch_size) << slog::endl;
+        printInputAndOutputsInfo(*model);
```

Comment:
Can we align naming style with OV? Maybe can we enable ncc check for samples?

Reply:
we'll do in separate PRs